internal/kafka/internal/migrations/migrations.go

```diff
@@ -88,7 +88,9 @@ var migrations = []*gormigrate.Migration{
 }
 
 func New(dbConfig *db.DatabaseConfig) (*db.Migration, func(), error) {
-	return db.NewMigration(dbConfig, &gormigrate.Options{
+	dbConfigWithDisabledPreparedStatements := dbConfig.DeepCopy()
+	dbConfigWithDisabledPreparedStatements.EnablePreparedStatements = false
```

Comment:
Thanks for the fix @miguelsorianod 

We could add a comment why this is disabled in the migration. Accompanied with the upstream created issue if possible. What do you think?

Reply:
Comment added.

I worked on creating the issue in the gorm project, including information about how to reproduce it and reproducible code included.

The issue is https://github.com/go-gorm/gorm/issues/5737

Reply:
Resolved. See https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pull/1314#issuecomment-1305714889